src/fluree/db/server_settings.clj

```diff
@@ -287,13 +288,13 @@
   (or (when-let [priv (:fdb-group-private-key env)] (str/trim priv))
       (let [priv-key-file (or (:fdb-group-private-key-file env)
                               "default-private-key.txt")
-            file          (io/file priv-key-file)
+            file          (io/file (:fdb-group-config-path env) priv-key-file)
             exists?       (.exists file)]
         (if exists?
-          (str/trim (slurp priv-key-file))
+          (str/trim (slurp file))
```

Comment:
@cap10morgan Can you explain 291 and 293 to me so I can learn? I get that you added the optional path, but why are you now slurping and spitting the file variable instead of the priv-key-file variable? Thanks

Reply:
@jakep36 Sure. It's because `file` contains the full path to the file (b/c of the change to line 290/291) whereas `priv-key-file` is just the filename.

Reply:
Ok. that makes sense. would it have worked to slurp file vs. priv-key-file in the existing code, too? Are they just the same in that case because they both point to a file in the current path?

Reply:
No, it wouldn't have worked. The heart of the change is line 290/291. That's what prefixes the path to the file to the filename (`clojure.java.io/file`'s signature is something akin to `(defn file [path filename] ...)`.